MapboxDirectionsTests/V5Tests.swift

```diff
@@ -69,6 +69,8 @@ class V5Tests: XCTestCase {
         let leg = route!.legs.first!
         XCTAssertEqual(leg.name, "I 80, I 80;US 30")
         XCTAssertEqual(leg.steps.count, 59)
+        let roadClasses = leg.steps.first!.intersections!.first!.roadClasses!
```

Comment:
Before `!` force-unwrapping `steps.first` and `intersections` and `intersections.first` and `roadClasses`, we should assert that each of these things are non-nil or non-empty. Otherwise we’ll have an unhelpful assertion message when the following assertion fails.